templates/imcore.yaml

```diff
@@ -0,0 +1,122 @@
+# Projects ----------------------------------------------------------
+
+imCORE Data Repository:
+  type: Project
+  acl:
+    # Question: Can you use the 'Ref' function in YAML keys?
```

Comment:
`!Ref` is not something that's built into standard yaml.  I believe the way to make it available is to create a custom yaml tag, https://stackoverflow.com/questions/43058050/creating-custom-tag-in-pyyaml